ArduCopter/Parameters.h

```diff
@@ -373,6 +373,9 @@ class Parameters {
 
         // 254,255: reserved
 
+        k_param_helispdhgtctrl = 256,
+        k_param_arot = 257,  // Autorotation controller library
+
         // the k_param_* space is 9-bits in size
```

Comment:
Per last night's Dev call, these will need to be moved to the G2 parameter class below.  Looks like the syntax is different for including the parameters in the group.

Reply:
Having a spot of trouble with this.  I have moved the parameters over to the G2 group and the code builds.  However, I cant see the parameters in MP.  I have pushed a commit with my changes.  Would appreciate it anyone could have a look over it to see if they can spot what I am doing wrong.

Reply:
This is now done.